webview/src/experiments/components/table/styles.module.scss

```diff
@@ -566,6 +566,19 @@ $bullet-size: calc(var(--design-unit) * 4px);
       outline-offset: -4px;
     }
   }
+
+  .otherCommitsRow {
```

Comment:
Currently the row has no hover animation but we could change the background color on hover like we do with other rows if preferred.

Reply:
no need for this I think